generators/craft/templates/env.sample

```diff
@@ -1,15 +1,18 @@
 APP_IMAGE_DRIVER="imagick"
```

Comment:
I don't think anything in this file should have changed.

Reply:
@cmalven I didn't actually change anything except the order of things. I just want all the database stuff gathered together, since we need to update that for every project.

Reply:
![screen shot 2017-10-23 at 3 22 49 pm](https://user-images.githubusercontent.com/872219/31911384-1687286a-b806-11e7-9b0e-f8d37ff32549.png)
Here's a more clear preview.

Reply:
@MikeMcChillin Still not very clear though because we have some of the imager configuration items stored in different places. This should also probably have been a separate PR so people can evaluate each of these changes as completely separate concerns (which they are).

Reply:
@cmalven Ah, good catch - I missed the if statement for imager. Reverted `env.sample` in this PR and opened new one for that update: https://github.com/onedesign/generator-one-base/pull/101

Reply:
Thanks @MikeMcChillin 

The only issue with this PR is that it means that If Asset Rev 5.5 (for instance) is available, we'll never grab that latest release when running the generator. We'll always get 5.0. I'm opening an issue on the Asset Rev repo to see if they can help us out with this before we merge this. If I don't hear back soon I'll merge.

https://github.com/clubstudioltd/craft-asset-rev/issues/16

Reply:
Sounds good @cmalven. I've also had good luck with this revving plugin hard-coded in `blendid` as an alternative solution: https://github.com/vigetlabs/blendid/tree/3a36e9f29a4ac564a8828693b16b771be01ffcd0/extras/craft/craft/plugins/gulprev